### PR TITLE
chore: Update docs

### PR DESCRIPTION
**Pull request by upbqdn**
## Motivation

Prepare for v 2.0

## Solution

- Update badges in readme.
- Simplify the description of Zebra in readme.
- Update "Known Issues".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,28 +2,30 @@
...
 - [Documentation](#documentation)
 - [User support](#user-support)
 - [Security](#security)
 - [License](#license)
 
 ## About
 
-[Zebra](https://zebra.zfnd.org/) is the Zcash Foundation's independent,
-consensus-compatible implementation of a Zcash node.
+[Zebra](https://zebra.zfnd.org/) is Zcash node written in Rust.
```
Comment (oxarbitrage) [inline README.md]:
```suggestion
[Zebra](https://zebra.zfnd.org/) is the Rust implementation of the Zcash blockchain.
```

**Review by oxarbitrage — DISMISSED** [review]
Thanks for the badge fixes! Looks good to me, i left a small suggestion but feel free to ignore it or change it.

**Review by oxarbitrage — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (upbqdn) [reply README.md]: I actually had a typo there. I updated it in [83c6e68](https://github.com/ZcashFoundation/zebra/pull/8970/commits/83c6e68f84e0cd769f11cfaab603b93a6c88a1ec).

I prefer the term "full-node" for Zebra, but feel free to change it the next time. :D

Comment (oxarbitrage) [reply README.md]: that's good enough for me. thanks!

**Pull request merged**